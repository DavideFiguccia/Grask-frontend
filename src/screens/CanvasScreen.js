import React from 'react';
import { GraphEditor } from '../components/graph_editor/GraphEditor';

export class CanvasScreen extends React.Component {

    render() {
        return <div>
            <GraphEditor/>
        </div>
    }

}


