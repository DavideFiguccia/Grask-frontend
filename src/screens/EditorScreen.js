import React from 'react';
import { GraphEditor } from '../components/GraphEditor/GraphEditor';
import { Container, Row, Col, Button } from 'react-bootstrap';
import ButtonRedux from '../components/ButtonRedux';

export function EditorScreen() {
	return (
		< Container fluid >
			<Row className="vh-100">
				<Col sm={2} className="px-0 border border-secondary">
					<Row className="mt-4 justify-content-center">
						<Button className="px-5" variant="outline-secondary">Carica</Button>{' '}
					</Row>
					<Row className="mt-2 justify-content-center">
						<ButtonRedux text="Nuovo"></ButtonRedux>
					</Row>
					<Row className="mt-2 justify-content-center">
						<Button className="px-5" variant="outline-success">Salva</Button>{' '}
					</Row>
				</Col>
				<Col sm={10}>
					<Container fluid>
						<GraphEditor />
					</Container>
				</Col>
			</Row>
		</Container >

	)

}


