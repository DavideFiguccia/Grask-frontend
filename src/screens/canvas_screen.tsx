import React from 'react';
import '../components/graph_editor/graph_editor.css';
import '../components/graph_editor/graph_editor';

export class CanvasScreen extends React.Component {

    render(){
        return <div className='graph_editor.css'>
            <script src="../components/graph_editor/graph_editor.js"/>
        </div>
    }

}


