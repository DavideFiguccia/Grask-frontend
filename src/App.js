import React from 'react';

import './App.scss';
import { EditorScreen } from "./screens/EditorScreen";
import { AppBar } from './components/AppBar/AppBar';
import { Footer } from "./components/Footer/Footer";
import { Container, Row, Col } from 'react-bootstrap';

function App() {
    return (
        <Container fluid>
            <Row>
                <Col className="px-0"><AppBar /></Col>
            </Row >
            <Row >
                <Col className="px-0">
                    <EditorScreen /> {/* All the pages here? */}
                </Col>
            </Row>
            <Row>
                <Col className="px-0"><Footer /></Col>
            </Row>
        </Container>
    );
}
export default App;
