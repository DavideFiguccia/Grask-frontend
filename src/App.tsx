import React from 'react';

import './App.css';
import {CanvasScreen} from "./screens/CanvasScreen";
import { AppBar } from './components/AppBar/AppBar';

function App() {
  return (
    <div className="App">
      
      <AppBar></AppBar>
      
      <div>
        <CanvasScreen/>
      </div>

    </div>
  );
}

export default App;
