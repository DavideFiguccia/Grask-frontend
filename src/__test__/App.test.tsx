import React from 'react';

import { create } from "react-test-renderer";


import App from '../App';
import { CanvasScreen } from '../screens/canvas_screen';

test("Matches the snapshot", () => {
    const button = create(<CanvasScreen />);
    expect(button.toJSON()).toMatchSnapshot();
  });
