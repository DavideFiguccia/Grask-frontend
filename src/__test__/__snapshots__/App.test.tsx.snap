// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
<div
  className="graph_editor.css"
>
  <script
    src="../components/graph_editor/graph_editor.js"
  />
</div>
`;
